rocAL/source/augmentations/geometry_augmentations/node_crop_resize.cpp

```diff
@@ -34,7 +34,11 @@ void CropResizeNode::create_node() {
         return;
 
     if (_is_random_crop) {
-        _crop_param->create_array(_graph);
+        if (_is_random_decode_crop) {
+            _crop_dec_param->create_array(_graph);
```

Comment:
Check if we can create one param and use for both

Reply:
This function requires returning a CropParam object so we cast the derived Param object class to its base class. We cannot use a single CropParam instance for this node since all three has different functionality and we cannot call derived class functions from Base class object